### ISSUE TITLE
Establish working email addresses for feedback hyperlinks

### ISSUE THREAD

**Issue by jimallman** [NESCent/FossilCalibrations#57]
The FCDB site currently uses <contact@calibrations.palaeo-electronica.org> as the email recipient in several `mailto:` hyperlinks. This should be enabled (or replaced with a real working email address) before launch. 

I've sent a test message to this address, which was rejected with this result:
```
Delivery to the following recipient failed permanently:

    contact@calibrations.palaeo-electronica.org

Technical details of permanent failure: 
DNS Error: Address resolution of calibrations.palaeo-electronica.org. failed: Domain name not found
```
FYI, here are the current `mailto:` links on the site:

 - In the site's header: [Contact Us](mailto:contact@calibrations.palaeo-electronica.org?subject=FCD%20feedback) 
 - The link above is repeated on the site's About page and again in the FAQ.
 - The calibration display page has a [Comment on this calibration](mailto:contact@calibrations.palaeo-electronica.org?subject=Comment%20on%20calibration%20XXX) link, which has a dynamic subject line based on the current calibration.

**Comment by Ksepka** [comment]
I just tried sending a test email using the "Comment on this Calibration" and "Contact Us" links.  Both attempt to send the message to:

contact@calibrations.palaeo-electronica.org

My gmail account reported:

DNS Error: Address resolution of calibrations.palaeo-electronica.org. failed: Domain name not found

I was under the impression we had fixed this - now I am not sure whether the address is misformed or something else is wrong.
